play/apps/snake/migrations/0001_initial.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.0.3 on 2018-06-25 03:16
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import util.fields
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Snake',
+            fields=[
+                ('id', util.fields.ShortUUIDField(max_length=128, primary_key=True, serialize=False)),
+                ('name', models.CharField(max_length=128)),
+                ('url', models.CharField(max_length=128)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='UserSnake',
+            fields=[
+                ('snake', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to='snake.Snake')),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
```

Comment:
Django is giving a warning on this field, it's strange though because I don't see a unique constraint provided here

```
snake.UserSnake.snake: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
```